### PR TITLE
LMS-172

### PR DESCRIPTION
**Pull request by OlehKosylo**
Fixed bug with delete question

### CODE REVIEW

File: src/middlewares/question/isQuestionPresent.middleware.ts [hunk 1]
```diff
@@ -7,9 +7,16 @@ import { questionService } from '../../services';
 
 export const isQuestionPresentMiddleware = async (req: IRequestExtended, res: Response, next: NextFunction) => {
   try {
-    const { question_id } = req.params;
-    const question = await questionService.getQuestionById(question_id);
+    const {question_id} = req.params;
+    if (!question_id) {
```
Comment (VictorKmin) [inline src/middlewares/question/isQuestionPresent.middleware.ts]: Will be better to use native mongo function
https://stackoverflow.com/questions/13850819/can-i-determine-if-a-string-is-a-mongodb-objectid

File: src/routes/question/question.router.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { Router } from 'express';
 import { questionController } from '../../controllers';
 import {
   checkAccessTokenMiddleware, checkIsTeacher, isQuestionFilterValid,
-  isQuestionIdValid, isQuestionPresentMiddleware, isQuestionValid,
+  isQuestionPresentMiddleware, isQuestionValid,
```
Comment (VictorKmin) [inline src/routes/question/question.router.ts:7]: Empty line after each import

File: src/middlewares/question/isQuestionPresent.middleware.ts (hunk 1 above)
Comment (OlehKosylo) [reply src/middlewares/question/isQuestionPresent.middleware.ts]: Done

File: src/routes/question/question.router.ts (hunk 2 above)
Comment (OlehKosylo) [reply src/routes/question/question.router.ts:7]: Done

**Review by VictorKmin — APPROVED** [review]

**Pull request merged**